### DISCUSSION TITLE
Keep asset index folder or subfolder (source) after saving an asset

### DISCUSSION THREAD

**Discussion by rocknrolaf** [craftcms/cms#9546] [Ideas] [answered]
When editing an asset within a subfolder and saving this asset, the redirect to the asset index looses the previously chosen subfolder (Craft version 3.6.17).
It should keep the subfolder/source like on the other element index views. This would make it easier to edit a bunch of related assets in one go (e.g. set alt texts for images etc.).

**Comment by brandonkelly** [comment, marked as answer]
Managed to get this into today’s 3.7.0 release ✨
